examples/create.rs

```diff
@@ -9,7 +9,7 @@ use std::io::Write;
 
 #[no_mangle]
 pub fn main() -> i32 {
-    let mut f = File::create(b"/target/hello\0").unwrap();
+    let mut f = File::create("target/hello").unwrap();
```

Comment:
You'll have to change this back to `/target/hello`. I know it looks wrong but because of how `cross` mounts directories on the docker image, it has to be like that.